src/index.js

```diff
@@ -20,15 +20,85 @@ function createElement(tagName, props, ...children) {
   return element;
 }
 
-function render() {
+function render(number, sum = 0, operator = '') {
```

Comment:
이렇게 매개변수가 많아질 때에는 순서나 값 등이 헷갈릴 수 있어서
매개변수를 객체 형태로 선언해서 사용하시면 코드 가독성을 높이는데 도움이 될거에요 :)

```suggestion
function render({ number, sum = 0, operator = '' }) {
```

Reply:
넵! 객체 형태로 선언하도록 하겠습니다 👍